src/main/java/br/com/dextra/dexboard/planilha/Planilha.java

```diff
@@ -80,7 +80,7 @@ protected Double recuperarConteudoCelulaDouble(int linha, int coluna) {
 
 	protected Double recuperarConteudoCelulaDouble(int linha, String nomeColuna) {
 		String value = recuperarConteudoCelula(linha, nomeColuna);
-		if (nomeColuna.equals("CPI") && value == null) {
+		if (nomeColuna.equals("CPI") && value == null || value.equals("")) {
```

Comment:
Frescura aqui, mas acho sempre legal colocar `()` pra explicitar a precedência das operações. Nem sempre acho tão óbvio (não é o caso, mas...) :stuck_out_tongue: 
[tópico no StackOverflow - Boolean operators precedence](https://stackoverflow.com/questions/12494568/boolean-operators-precedence#answer-52904047)

```suggestion
		if ((nomeColuna.equals("CPI") && value == null) || value.equals("")) {
```

Reply:
vou alterar, mas acho q deveria ser assim:
`if (nomeColuna.equals("CPI") && (value == null || value.equals(""))) {`
será q não?

Reply:
beleza

Reply:
só atualizando, na verdade o comportamento esperado era:
`if (nomeColuna.equals("CPI") && (value == null || value.equals(""))) {`
Do jeito q estava, estava errado